docs/application/native/guides/text-input/tts.md

```diff
@@ -340,6 +340,39 @@ To set and unset callbacks, follow these steps:
      }
      ```
 
+   - Synthesized PCM
+
+     When a synthesized PCM data is received from a TTS service, the TTS library sends a synthesized PCM data using a callback:
+
+     ```cpp
+     /* Callback */
+     void
+     synthesized_pcm_cb(tts_h tts, int utt_id, tts_synthesized_pcm_event_e event, const char* pcm_data, int pcm_data_size, tts_audio_type_e audio_type, int sample_rate, void *user_data);
```

Comment:
```suggestion
     synthesized_pcm_cb(tts_h tts, int utt_id, tts_synthesized_pcm_event_e event, const char* pcm_data, int pcm_data_size, tts_audio_type_e audio_type, int sample_rate, void *user_data)
```

Reply:
Thank you for checking. I've just fixed it.